requirements-ci.txt

```diff
@@ -1,4 +1,4 @@
 -r requirements.txt
 -r requirements-tests.txt
-libtorrent==2.0.7
-pytest==7.4.2
+pytest
```

Comment:
Currently, pytest is pinned due to console tests failing, which is fixed by this PR.